core/types/rollup_cost.go

```diff
@@ -25,21 +26,34 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+const (
+	// offsets of the packed fields within the L1FeeScalarsSlot
+	BasefeeSlotOffset     = 0
+	BlobBasefeeSlotOffset = 4
+)
+
+var (
+	BedrockL1AttributesSelector = []byte{0x01, 0x5d, 0x8e, 0xb9} // function selector indicating Bedrock style L1 gas parameters
+	EcotoneL1AttributesSelector = []byte{0x44, 0x0a, 0x5e, 0x20} // function selector indicating Ecotone style L1 gas parameters
```

Comment:
checked these, but would be nice to add a test assertion in the monorepo Go code

Reply:
ack